src/board/board.service.ts

```diff
@@ -29,16 +27,42 @@ export class BoardService {
   }
 
   async create(userId: number, dto: CreateBoardDto): Promise<board> {
-    const [boards] = await this.prisma.$transaction([
-      this.boardsRepository.create(userId, dto),
-      this.categoriesRepository.multiCreate(dto.categories),
-    ]);
-    const categories = await this.categoryService.getCategoriesByNames(
-      dto.categories,
-    );
-    const categoryIds = categories.map((category) => category.id);
-    await this.categoryOnBoardService.multiCreate(categoryIds, boards.id);
-    return boards;
+    return await this.prisma.$transaction(async (prisma) => {
+      const board = await prisma.board.create({
+        data: {
+          title: dto.title,
+          context: dto.context,
+          thumbnail: dto.thumbnail !== null ? dto.thumbnail : 'thumbnail',
```

Comment:
```suggestion
          thumbnail: dto.thumbnail ?? 'thumbnail',
```
dto.thumbnail이 null과 유효한 경로 두 가지로 주어진다면 `??` 를 사용해서 명시적으로 작성해도 좋을 거 같아요!